.gitignore

```diff
@@ -3,3 +3,7 @@
 **/Session.vim
 cnndm_acl18
 cnn_stories_tokenized/finished_files*
+cnn_stories_tokenized/1000_test
+corpus.txt
+vocab.txt
```

Comment:
shouldn't this be in the repo?

Reply:
I can add in the 1000_test file but the corpus is a very large file (375mb) and the vocab doesn't help that much without the corpus

Reply:
added commit for 1000_test below